packages/yourloops/models/team.ts

```diff
@@ -51,7 +51,8 @@ export interface ITeamMember {
   settings?: Settings | null;
   preferences?: Preferences | null;
   idVerified: boolean;
-  alert?: Alarm;
+  alarms?: Alarm;
```

Comment:
maybe | null missing here ?

Reply:
I don't know anymore, if it is ok with you I'd like to keep it as is